website/blog/2024-04-08-connect-private-github-repository-to-tabby/index.md

```diff
@@ -42,7 +42,7 @@ name = "another_project"
 git_url = "https://<PAT>@github.com/icycodes/another_project.git"
 ```
 
-For more detailed about the configuration file, you can refer to the [configuration documentation](https://tabby.tabbyml.com/docs/configuration).
+For more detailed about the configuration file, you can refer to the [configuration documentation](https://tabby.tabbyml.com/docs/extensions/configurations/).
```

Comment:
```suggestion
For more detailed about the configuration file, you can refer to the [configuration documentation](https://tabby.tabbyml.com/docs/configuration).
```

The original configuration file in the repository was removed in past. A newer update will soon be available after version 0.12.